_includes/author.html

```diff
@@ -0,0 +1,13 @@
+<div class="ui card">
+	<div class="bottom aligned content">
+		<div class="center aligned segment">
+			<img class="ui image circular" alt="Photo de {{ include.description.fullname }}" src="img{{include.description.id}}.jpg" onError="this.src = 'img/author/egg.jpg';" style="width: 150px; height:auto;"/>
```

Comment:
"Egg" en référence à l'icône de profil anonyme de Twitter: mon intention c'est de remplacer à terme le tube à essai par une image plus pertinente (genre http://fanreact.com/wp-content/uploads/2014/09/21738698-male-silhouette-profile-picture-with-question-mark-on-the-head-vector10.jpg), donc je l'ai "découplée" du logo incubateur.